### DISCUSSION TITLE
The content of the page is not saved properly

### DISCUSSION THREAD

**Discussion by mukhammart** [requarks/wiki#7377] [Error / Bug Report]
Whenever I update the content in "Markdown mode" and press "Save", and close the window of my browser and enter the window again, the content is not saved. It's saved only when I press "Save" and after that "Close".

I guess it is a bug, looking forward to fix, thank you in advance!

**Comment by NGPixel** [comment]
I can't replicate. Make sure your browser doesn't cache the page?
